pandas/tests/indexes/base_class/test_repr.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+from pandas import Index
+
+
+class TestIndexRepr:
```

Comment:
rather than create a new file, can you put this in pandas/tests/indexes/test_base.py its where these tests currently exist (separataly we can certain move these, but they need to move all together)